Travis-ci: Added support for ppc64le build along with amd64.

sanjaymsh opened the pull request:
Hi,
I have added support for ppc64le build on travis-ci in the branch . The travis-ci build log can be tracked on the link :https://travis-ci.com/github/sanjay-cpu/postgresql-setup . I believe it is ready for the final review and merge.
Please have a look on it and if everything looks fine for you then please approve it for merge.

Thanks !!

sanjaymsh committed:
Travis-ci:added support for ppc64le along with amd64

The -t flag is removed because it is not supported on ppc64le archictecture on travis and throws a error "The input device is not a TTY"

sanjaymsh commented:
Hi @praiskup  Can you please look into this PR? Thanks !!

praiskup commented on .travis/run-in-docker:
```diff
@@ -17,8 +25,12 @@ run_in_container ()
-
+if [ "$ARCH" = "ppc64le" ]; then
+    docker exec -u $uid -i "$container_id" "$@"
+else
```
Why we can not use `-t` here, too?

sanjaymsh replied:
Hi @praiskup , I removed '-t' from here because it was throwing error like "The input device is not a TTY". 

Thanks for review and comment.

praiskup replied:
Ok, it deserves a comment above the line ... and perhaps we could drop `-t` for all architectures?

sanjaymsh replied:
Yes, we can remove it and i have done that as well. Thanks !!

praiskup commented:
Thank you for the pull request, @sanjay-cpu.  Maintainers of this repository are @pkubatrh, @hhorak and @UncleAlbie,
but I tried to look at the code a bit and it looks fine.

The main problem I see now is that Travis doesn't spawn builds for some reason, even though it looks it is enabled.

praiskup commented:
Travis CI run fine, I just don't see Travis updating the badge (build status) here.
https://travis-ci.org/github/devexp-db/postgresql-setup/builds/719862036

praiskup commented:
Just please update the commit message so it looks like: "travis: add a ppc64 build along the amd64".  From the subject, it's not clear that the commit is only related to Travis.

praiskup commented:
Ok, seems to be fixed.  I had to install Travis CI App to the @devexp-db organization, and re-sync && re-enable travis on travis.org side.

sanjaymsh commented:
> Just please update the commit message so it looks like: "travis: add a ppc64 build along the amd64".  From the subject, it's not clear that the commit is only related to Travis. 

I have updated the title. Thanks!!

praiskup commented:
Please update the commit message `git commit --amend`.

sanjaymsh commented:
Hi @praiskup , Thanks for four feeadback.  I have updated the commit message and removed the "-t" flag for all the archs as suggested. Now it's ready for the final review and merge. 

Thanks !!

praiskup commented:
@sanjay-cpu thank you!
